CredentialProvider/View/CredentialItemListView.swift

```diff
@@ -23,4 +23,38 @@ class ItemListView: BaseItemListView, ItemListViewProtocol {
     override func createPresenter() -> BaseItemListPresenter {
         return ItemListPresenter(view: self)
     }
+
+    override func styleNavigationBar() {
+        super.styleNavigationBar()
+
```

Comment:
I tried putting `self.navigationController?.navigationBar.tintColor = UIColor.white` here but it didn't work. Sorry I couldn't be helpful.